README.md

```diff
@@ -109,4 +109,10 @@ array([[2.16105225e-04, 5.58412459e-04, 9.70510737e-05, 5.76671700e-05,
 * TopK
```

Comment:
`LSTM` is supported. So does `ArgMax` and `ArgMin`.

Reply:
Not in Python version, which is our current official version.